src/state/user/actions.ts

```diff
@@ -38,3 +38,6 @@ export const updateUserFarmStakedOnly = createAction<{ userFarmStakedOnly: FarmS
   'user/updateUserFarmStakedOnly',
 )
 export const updateGasPrice = createAction<{ gasPrice: string }>('user/updateGasPrice')
+
+export const addWatchlistToken = createAction<{ address: string }>('user/addWatchlistToken')
```

Comment:
Update to existing code:
Couple new actions for `user` reducer - adding tokens/pools to watchlist. Here cause user state persists (and it just makes more sense)